openpathsampling/engines/dynamics_engine.py

```diff
@@ -262,8 +268,8 @@ def __getattr__(self, item):
                     # alternately, trust the fixed result with
                     # return result  # miraculously fixed
                     raise AttributeError(
-                        "Unknown problem occurred in property" + 
-                        str(p.fget.func_name) + ": Second attempt returned"
+                        "Unknown problem occurred in property"
+                        + property_name(p) + ": Second attempt returned"
```

Comment:
See other comment: this should be replaced with the 2.7/3.x compatible `str(p.fget.__name__)` (may not require `str`? can't hurt though)